ledfx/core.py

```diff
@@ -160,6 +163,13 @@ def handle_visualisation_update(event):
             if len(pixels) > max_len:
                 pixels = interpolate_pixels(pixels, max_len)
 
+            if self.config["transmission_mode"] == 1:
+                b_arr = bytearray()
+                b_arr = bytes(pixels.astype(np.uint8).flatten())
+                pixels = base64.b64encode(b_arr).decode("ASCII")
+            elif self.config["transmission_mode"] == 0:
```

Comment:
In theory this can be an else, since current state it can't be anything other than 0/1